### ISSUE TITLE
Tiles need to update on range slider action

### ISSUE THREAD

**Issue by peterqliu** [mapbox/mapbox.js#911]
From the [demo](https://www.mapbox.com/mapbox.js/example/v1.0.0/swipe-layers/):

![range slider](https://cloud.githubusercontent.com/assets/4551508/4986651/9524fabe-693d-11e4-8cb9-ccd2fa2633bf.gif)

They do update when I zoom, of course, but I also need it when the slider moves, to maintain the desired illusion.

**Comment by mtirwin** [comment]
Just ran into a similar issue with the slider while building an Atlas demo, though I'm only seeing intermittent updates when I zoom. I'm running Chrome on Yosemite, and the issue is replicable whether hitting the Mapbox.com API w/ online mapids or the Atlas API with offline mapids.

![brokenfade](https://cloud.githubusercontent.com/assets/7843542/5152542/ce9668fc-71c4-11e4-9294-1d18cb1afc98.gif)

@mourner [this old issue](https://github.com/mapbox/mapbox.js/issues/168) is the only thing I found referencing trouble with the Leaflet slider. Any sense of what @peterqliu and I are encountering?

**Comment by mourner** [comment]
@mtirwin @peterqliu what are your browsers? The mapbox.js example is working fine for me in Chrome & Firefox under OS X.

**Comment by mtirwin** [comment]
@mourner, I'm using Chrome version 38.0.2125.122 (current) on OSX Yosemite. The issue doesn't replicate on Safari.

**Comment by mourner** [comment]
OK, managed to reproduce this, although it's happening sporadically... My guess is that it's a Chrome bug.

**Comment by peterqliu** [comment]
@mourner yes, I just updated my Chrome and the problem disappeared for me. Should we close?

**Comment by mourner** [comment]
Yeah, lets close — should go away with a Chrome update hopefully.

**Comment by jfirebaugh** [comment]
FWIW, this is still happening for me in latest Chrome (38.0.2125.122).

**Comment by peterqliu** [comment]
Yup, I'm seeing it again-- seems entirely sporadic. Reopened.

**Comment by jfirebaugh** [comment]
@peterqliu Can you make a self contained example and submit an issue on https://code.google.com/p/chromium/issues/list?

**Comment by mtirwin** [comment]
@peterqliu did you submit an issue on Chromium? Noticing our slider's still broken on the latest version of Chrome (39.0.2171.95).

**Comment by peterqliu** [comment]
@mtirwin I actually haven't submitted (my bad), but the bug has disappeared for me again. Would you mind submitting, since you are still experiencing this?

**Comment by lxbarth** [comment]
I confirm this bug on Mac OSX, Chrome version 43.0.2357.124 (64-bit)

**Comment by lxbarth** [comment]
> @peterqliu Can you make a self contained example and submit an issue on https://code.google.com/p/chromium/issues/list?

@peterqliu - would love to see this happen. Here's an animated gif from my 43.0.2357.124

![swipe](https://cloud.githubusercontent.com/assets/98233/8628921/a36d1174-270b-11e5-85be-041dc8976c0e.gif)

**Comment by lxbarth** [comment]
I filed a bug w/ the Chrome project: https://code.google.com/p/chromium/issues/detail?id=509126&thanks=509126&ts=1436570972

The issue is that setting the clip style attribute does not force a repaint. Not sure why this only occurs after zooming once.

And here is a workaround that fixes the issue for me on Chrome 43.0.2357.132 (64-bit) (from [stackoverflow #3485365](http://stackoverflow.com/questions/3485365/how-can-i-force-webkit-to-redraw-repaint-to-propagate-style-changes)):

```
elem.style.display='none';
elem.offsetHeight;
elem.style.display='';
```

Workaround in action: http://bl.ocks.org/lxbarth/bcd0d1ab3c69085e2dad

@tmcw - worthwhile getting the workaround into the example code?
